db/migrate/20171215154809_create_geo_monitor_layers.geo_monitor.rb

```diff
@@ -0,0 +1,16 @@
+# This migration comes from geo_monitor (originally 20171212180244)
+class CreateGeoMonitorLayers < ActiveRecord::Migration[5.1]
```

Comment:
why do we need single-field indexes on `access` and `checktype`?

Reply:
So this was coming upstream from GeoMonitor. At the moment I'm not sure we do need these, but it may be helpful (perhaps?) if we need to query by them at some point. Would you prefer we remove them to begin with?

Reply:
yeah i would take them out and tune for performance later. usually you need composite keys for status-like fields for them to be used efficiently -- i.e., `where(access: 'Public', checktype: 'whatever')` benefits more from a composite key on access + checktype.

Reply:
👍